from_cpython/Objects/stringobject.c

```diff
@@ -23,6 +23,22 @@ PyObject * _do_string_format(PyObject *self, PyObject *args, PyObject *kwargs) {
     return do_string_format(self, args, kwargs);
 }
 
+void _string_init() {
```

Comment:
Do we have to to do this?
Looks like cpython does it in ```_PyUnicode_Init``` and we use this one do so I think we can remove it.